src/redux/books/books.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const bookSlice = createSlice({
+  name: 'book',
+  initialState: [],
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.push(payload);
+    },
+    removeBook: (state, { payload }) => state.filter(({ id }) => id !== payload),
+  },
+});
+
+export const { addBook, removeBook } = bookSlice.actions;
+
+export default bookSlice.reducer;
```

Comment:
- [ ] as part of the requirements, The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:  
  `const ADD_BOOK = bookstore/books/ADD_BOOK` 
  `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

- [ ] and then you need to create a reducer that checks the action type using case, for example : 

```JS
 switch (action.type) {
    case ADD_BOOK :
     ....
    case REMOVE_BOOK:
     ....
    default:
     ....
  }
```

kindly fix this for both books & categories 🙏